### PR TITLE
Add gem Ransack

### PR DESCRIPTION
**Pull request by khanhnp-2797**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/44432

## WHAT (optional)
- Add gem Ransack

## Evidence (Screenshot or Video)
Screenshot from 2021-12-21 09-26-00

### CODE REVIEW

**Comment by khanhnp-2797** [comment]
ready

File: app/controllers/admin/products_controller.rb [hunk 1]
```diff
@@ -81,12 +85,7 @@ def load_product
...
-    @order_detail = OrderDetail.find_by(product_id: @product.id)
-  end
-
-  def search
-    @products = Product.find_name(params[:name])
-                       .search_category(params[:category_id])
-  end
+  # def load_order_details
+  #   @order_detail = OrderDetail.find_by(product_id: @product.id)
+  # end
```
Comment (ghost) [inline app/controllers/admin/products_controller.rb]: Sao lại comment hàm này em

File: app/views/admin/orders/index.html.erb [hunk 2]
```diff
@@ -6,6 +6,20 @@
     </div>
   </div>
 </div>
+<%= search_form_for [:admin,@q] do |f| %>
+  <div class="row">
+     <div class="col-5 lable-form">
+      <%= f.label t "search_admin.create_at_gteq" %>
+      <%= f.date_field :created_at_gteq, value: "%Y/%D/%M", class: "form-control" %><br>
+ </div>
```
Comment (ghost) [inline app/views/admin/orders/index.html.erb]: indent

**Review by ghost — COMMENTED** [review]
cmt

File: app/controllers/admin/products_controller.rb (hunk 1 above)
Comment (khanhnp-2797) [reply app/controllers/admin/products_controller.rb]: e quên mở ra anh :v

**Comment by khanhnp-2797** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**